examples/shared_vpc/outputs.tf

```diff
@@ -0,0 +1,47 @@
+/**
```

Comment:
For the sake of consistency we add descriptions to all outputs; let's get those added here. You can copy/paste most descriptions from `examples/simple_example`.

Reply:
fixed. thank you.